setup.py

```diff
@@ -178,6 +178,8 @@ def version(self):
 extra_reqs = {
     'test': ['pytest', 'pytest-cov', 'numpy>=1.4.1', 'packaging']
 }
+if os.getenv('NUMPY') == '0':
+    extra_reqs['test'].remove('numpy>=1.4.1')
```

Comment:
@snorfalorpagus instead, how about we add an extra requirements item: `'vectorized': ['numpy>=1.4.1']`? If we've already installed numpy, then nothing changes.